custom/auth/ante/expected_keeper.go

```diff
@@ -10,6 +10,7 @@ type TreasuryKeeper interface {
 	GetTaxRate(ctx sdk.Context) (taxRate sdk.Dec)
 	GetTaxCap(ctx sdk.Context, denom string) (taxCap sdk.Int)
 	GetBurnSplitRate(ctx sdk.Context) sdk.Dec
+	HasWhitelistAddress(ctx sdk.Context, address string) bool
```

Comment:
Would it maybe make sense to change the "address" param to a sdk.Address type? We should  be able to handle the casting in the msg server that processes the inbound msg types. :)

Reply:
I observe that string is used in most case so it would be more convenient to use string here